create_infotexts.py

```diff
@@ -471,15 +513,14 @@ def generate_list_of_stripped_keywords(self, keyword_string):
         :return: list of keywords.
         """
         keywords_list = keyword_string.split(", ")
-        keywords_list_low = [kw.lower() for kw in keywords_list]
```

Comment:
Actually, when I looked at the original metadata again I noticed that it was all lower case. Thus, I reduced complexity by removing this and gives the same end result.